cli/wallet/wallet.go

```diff
@@ -289,6 +301,55 @@ func claimGas(ctx *cli.Context) error {
 	return nil
 }
 
+func changePassword(ctx *cli.Context) error {
+	wall, err := openWallet(ctx.String("wallet"))
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+	addrFlag := ctx.Generic("address").(*flags.Address)
+	if addrFlag.IsSet {
+		// Check for account presence first before asking for password.
+		acc := wall.GetAccount(addrFlag.Uint160())
+		if acc == nil {
+			return cli.NewExitError("account is missing", 1)
+		}
+	}
+
+	oldPass, err := input.ReadPassword("Enter password > ")
+	if err != nil {
+		return cli.NewExitError(fmt.Errorf("Error reading old password: %w", err), 1)
+	}
+
+	for i := range wall.Accounts {
+		if addrFlag.IsSet && wall.Accounts[i].Address != addrFlag.String() {
+			continue
+		}
+		err := wall.Accounts[i].Decrypt(oldPass, wall.Scrypt)
+		if err != nil {
+			return cli.NewExitError(err, 1)
```

Comment:
Maybe add a specific error with the account address?

Reply:
As you wish.